.github/workflows/ci.yaml

```diff
@@ -72,7 +72,7 @@ jobs:
           fi
         done
         source ./tests/scripts/.definitions.sh
-        ./tests/scripts/pull.sh ${LOG_DIR} logs
+        ./tests/scripts/pull.sh /tmp/logs logs
```

Comment:
Question -- if we reach this point, will there be any logs at `/tmp/logs`? My understanding is we only collect logs, by running `must-gather.sh`, only when a test case fails.

Reply:
Just to satisfy the artifacts step, if I put it inside an if condition, the artifacts step will fail with no path : ./logs